src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -289,7 +290,7 @@ export const ComparisonSide = ({
   };
 
   const onWebGLInitialized = (gl) => {
-    const stats = gl.stats.get("Memory Usage");
+    const stats = lumaStats.get(`Memory Usage${gl.canvas.id}`);
```

Comment:
Does it work now or we need wait for luma update?

Reply:
I guess we know this id without having a gl context. It should be `id` of `DeckGL` component